javascript/core/array-methods/filtering-items-out-of-an-array.md

```diff
@@ -29,65 +20,126 @@ aspects:
   - introduction
   - workout
 
+links:
+  - '[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter){documentation}'
 
 ---
 
-# Filtering items out of an array
+# Filtering Items out of an Array
 
 ---
 ## Content
 
 A common pattern to filter items from an array is the following:
 
-```
+```js
 let persons = [...];
 let activated = [];
-for(let person of persons) {
+for (let person of persons) {
   if (person.isActived) {
     activated.push(person);
   }
 }
 ```
-However, using the `Array::filter` method is generally more concise and readable:
 
+However, using the `Array.prototype.filter` method is generally more concise and readable. With `filter()`, you **create a new array** whose elements must pass a test. Here's the simplified syntax[1]:
+
+```js
+let newArray = arr.filter(
+  callback(element)
+);
 ```
-let persons = [...];
-let activated = persons.filter(
-  person => person.isActived
+
+You pass a callback function which is applied to every element, and if the result of `callback` is `true` then the element gets added to `newArray`. This might seem a bit confusing, so let's take a look at an example:
```

Comment:
```suggestion
You pass a callback function which is applied to every element, and if the result of `callback` is truthy then the element gets added to `newArray`. This might seem a bit confusing, so let's take a look at an example:
```